test/test_lshforest.py

```diff
@@ -62,6 +62,14 @@ def test_query(self):
             results = forest.query(data[key], 10)
             self.assertIn(key, results)
 
+    def test_get_minhash_hashvalues(self):
+        forest, data = self._setup()
+        for key in data:
+            minhash_ori = data[key]
+            hashvalues = forest.get_minhash_hashvalues(key)
+            minhash_retrieved = MinHash(hashvalues=hashvalues)
+            self.assertEqual(minhash_retrieved.jaccard(minhash_ori), 1.0)
```

Comment:
One more paranoid check to avoid floating point errors: compare the integer hash values directory.

Reply:
@ekzhu just pushed this change